.github/workflows/sonar_analysis_kiegroup.yml

```diff
@@ -0,0 +1,36 @@
+name: "SonarCloud Analysis: kogito-editors-java"
```

Comment:
why do we have both GHAction and Jenkins for Sonar ?
I think we should focus on one or ?

Reply:
yes it should as it is the default repository used (from getJobParams)

Reply:
I followed approach from the optaplanner, where
- [sonarcloud.yml](https://github.com/kiegroup/optaplanner/blob/main/.github/workflows/sonarcloud.yml) defines job to check each push to main branch by sonar
- [Jenkinsfile](https://github.com/kiegroup/optaplanner/blob/main/.ci/jenkins/Jenkinsfile) defines job to check each opened PR agains main branch by sonar

Should I use just Jenkinsfiles like we do for kogito-runtimes?

Reply:
oh my bad, I misread. so yeah it is ok :)